examples/form_demo/local/lib/form_demo_local.ex

```diff
@@ -66,6 +67,37 @@ defmodule FormDemoLocal do
     end
   end
 
+  def handle_event(
+        "llv_server_message",
+        %{"type" => "synchronize", "users" => server_users},
+        socket
+      ) do
+    filtered_users =
+      Enum.filter(socket.assigns.users, fn user ->
+        case validate_already_existing(user, server_users) do
+          [] ->
+            send_to_phoenix(%{"type" => "new_user", "user" => user})
```

Comment:
Similarly to sending a message from server to the browser - imo this should also be wrapped in some kind of function similar for LiveView users (like `send_update/3`). But this is a matter of crearing a proper API for LocalLiveViews